internal/mqini/mqini_test.go

```diff
@@ -59,6 +62,60 @@ func TestGetQueueManager(t *testing.T) {
 			if d != table.errorLogDir {
 				t.Errorf("Expected error log directory=%v; got %v", table.errorLogDir, d)
 			}
+
```

Comment:
It seems to me that this is really a few different tests conflated into one.  I'm finding it hard to see if you actually check that the resulting INI file has the right data in it.  By breaking up the code into smaller functions, I wonder if we can have separate Test functions which target the individual units more clearly.

Reply:
Thanks, I have rewritten tests to separate them and run as stanza verification tests and also increased coverage.